lib/pubsub/index.js

```diff
@@ -315,14 +323,40 @@ PubSub.prototype.subscription = function(name, options) {
  *
  * @param {string} name - The name of the topic.
  * @param {object=} options - Configuration object.
- * @param {boolean=} options.autoCreate - Automatically create topic if it
+ * @param {boolean} options.autoCreate - Automatically create topic if it
  *     doesn't exist. Note that messages published to a topic with no
- *     subscribers will not be delivered.
+ *     subscribers will not be delivered. Default: true.
  * @return {module:pubsub/topic}
  *
  * @example
- * var topic = pubsub.topic('my-existing-topic');
- * var topic = pubsub.topic('topic-that-maybe-exists', { autoCreate: true });
+ * //-
+ * // By default, it isn't required to specify a topic that already exists. The
+ * // first time you publish a message, the topic will be created for you.
+ * //
+ * // This will only cost one additional API request at the time of publishing.
+ * // If the topic doesn't need to be created, there is no performance penalty.
+ * //-
+ * var topic = pubsub.topic('my-topic');
+ *
+ * topic.publish({
+ *   data: 'New message!'
+ * }, function(err) {});
+ *
+ * //-
+ * // If you prefer an error when trying to interact with a topic that doesn't
+ * // exist, you can specify `autoCreate: false`.
+ * //-
+ * var nonExistentTopic = pubsub.topic('my-non-existent-topic', {
+ *   autoCreate: false
+ * });
+ *
+ * nonExistentTopic.publish({
+ *   data: 'New message!'
+ * }, function(err) {
+ *   if (err) {
+ *     // API error from trying to publish a message to a non-existent topic.
+ *   }
+ * });
```

Comment:
Question: If I don't create a topic because I don't have any messages to publish yet, and instead I subscribe to that topic assuming it already exists (as this would suggest I do), then will it throw an error in that case saying the topic doesn't exist?